ansible_collections/arista/avd/roles/eos_designs/docs/how-to/custom-structured-configuration.md

```diff
@@ -68,20 +68,20 @@ ip name-server vrf MGMT 192.168.42.40
 # Only ip_name_servers from eos_ci_config_gen.
 # The variables will make it to the intended config
 ip_name_servers:
-  - ip_address: 8.8.8.8
+  - ip_address: 1.1.1.1
     vrf: EOS_CLI
-  - ip_address: 4.4.4.4
+  - ip_address: 192.168.42.10
     vrf: EOS_CLI
 ```
 
 will generate as intended config:
 
 ```eos
-ip name-server vrf EOS_CLI 4.4.4.4
-ip name-server vrf EOS_CLI 8.8.8.8
+ip name-server vrf EOS_CLI 1.1.1.1
+ip name-server vrf EOS_CLI 192.168.42.10
 ```
 
-#### eos_cli_config_gen variables overwritten by eos_designs variables
+#### eos_cli_config_gen variables and eos_designs variables
```

Comment:
The body still says that eos_cli_config_gen variables overwritten by eos_designs variables so can we keep the heading same.

Reply:
Fixed